components/admin_console/reset_password_modal/reset_password_modal.tsx

```diff
@@ -33,7 +34,7 @@ type Props = {
     onModalDismissed: () => void;
     passwordConfig: PasswordConfig;
     actions: {
-        adminResetPassword: (userId: string, currentPassword: string, password: string, success: () => void, error: (err: Error) => void) => void;
+        updateUserPassword: (userId: string, currentPassword: string, password: string) => ActionFunc;
```

Comment:
By the time `updateUserPassword` has been passed into this component, it's been wrapped with dispatch, so it's returning the result of the `ActionFunc`. The new return type will be something like `{data?: <some type>, error?: Error}`